File: BC - w2/arrayScan.c
#include <stdio.h>

int arrayScan(FILE *in, int array[], int limit) {
    int i;
    for ( i = 0; i != limit && fscanf(in, "%d", &array[i]) == 1; i++ );
    return i;
}



int main(){
    FILE *in = fopen ("task.in", "r");

    int limit = 8;
    int array[limit];
    printf("%d\n", arrayScan(in, array, limit));

    fclose(in);

    return 0;
}

File: BCW5/PHP/arrayScan.php
<?php
    function arrayScan($file, $array, $limit) {
         $array =  str_split (fread($file, $limit));
         return count($array) ;
    }



    $file = fopen("Task.in", "rb");

    $array = array();
    $limit = 4;

    echo arrayScan($file, $array, $limit);
    fclose($file);

?>